port/common/omrmem32helpers.c

```diff
@@ -438,15 +430,28 @@ allocate_memory32(struct OMRPortLibrary *portLibrary, uintptr_t byteAmount, cons
 #endif
 		omrthread_monitor_enter(PPG_mem_mem32_subAllocHeapMem32.monitor);
 
-		/* Check if byteAmount is larger than HEAP_SIZE_BYTES.
+		/* Check if byteAmount is larger than PPG_mem_mem32_subAllocHeapMem32.suballocator_incrementSize.
 		 * The majority of size requests will typically be much smaller.
 		 */
 		returnPtr = iterateHeapsAndSubAllocate(portLibrary, byteAmount);
 		if (NULL == returnPtr) {
-			if (byteAmount >= HEAP_SIZE_BYTES) {
-				returnPtr = allocateLargeRegion(portLibrary, byteAmount, callSite, 0);
+			if (byteAmount >= PPG_mem_mem32_subAllocHeapMem32.suballocator_incrementSize) {
+				returnPtr = allocateLargeRegion(portLibrary,
+								byteAmount,
+								callSite,
+								0);
 			} else {
-				returnPtr = allocateRegion(portLibrary, HEAP_SIZE_BYTES, byteAmount, callSite, 0);
+#if defined(OMR_ENV_DATA64) && defined(LINUX)
+				/* For 64 bit linux, use the OMRPORT_VMEM_ALLOC_QUICK flag if it has not been disabled. */
+				uintptr_t vmemAllocOptions = PPG_mem_mem32_subAllocHeapMem32.suballocator_quickAlloc ? OMRPORT_VMEM_ALLOC_QUICK : 0;
+#else /* defined(OMR_ENV_DATA64) && defined(LINUX) */
+				uintptr_t vmemAllocOptions = 0;
+#endif /* defined(OMR_ENV_DATA64) && defined(LINUX) */
+				returnPtr = allocateRegion(portLibrary,
```

Comment:
First parameter should be on a new line: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line

Reply:
> The cost of this field existence and extra checks for platforms don't support the feature but it is ok for me.

Ok. I was just being on the conservative side here, but I can remove them if this is the sentiment from the team. I don't feel strongly about either option.